src/components/panels/ToolheadControls/CrossControl.vue

```diff
@@ -377,7 +377,16 @@ export default class CrossControl extends Mixins(BaseMixin, ControlMixin) {
     mdiHome = mdiHome
 
     get actionButton(): string {
-        return this.$store.state.gui.control.actionButton ?? this.defaultActionButton
+        const button = this.$store.state.gui.control.actionButton ?? this.defaultActionButton
+
+        if (
```

Comment:
combining the two ifs doesn't change the fact the same logic is used in both Controls `CrossControl.vue` and `CircleControl.vue` :D
But fine we can leave it like this